src/ethereum_spec_evm_resolver/forks.py

```diff
@@ -179,3 +178,20 @@ def get_fork_resolution(fork_name: str) -> Resolution:
         return default_resolutions[fork_name]
     else:
         raise Exception(f"Unable to resolve fork: {fork_name}")
+
+
+def get_fork_resolution_info(fork: str) -> Dict[str, str]:
+    fork_resolution = get_fork_resolution(fork)
+    try:
+        resolver_path = Path(platformdirs.user_cache_dir("ethereum-spec-evm-resolver"))
+        info_file = resolver_path / (f"{fork}.info")
+        git_resolution_info = GitResolutionInfo.model_validate_json(
+            info_file.read_text()
+        )
+        return {
+            "git-url": str(git_resolution_info.resolution.git_url),
+            "branch": git_resolution_info.resolution.branch,
+            "commit": git_resolution_info.resolution.commit or git_resolution_info.head,
```

Comment:
I figured having a consistent `commit` here and either point to the `head` if no commit was provided in the eels resolution file, or using the `commit` value that was provided would be better than switching `commit` with `head` here. But if it's better for a reader of the generated test to distinguish when no commit was explicitly given in the resolution file, we should probably make it quicker to read and understand that it's at the `head` and not at some other commit here.

Thoughts on that?

Reply:
I agree with current behavior, I think it's better to always show the commit here regardless of how it was specified in the eels resolution file IMO.

Reply:
Agree, the end user only cares which precise version of `t8n` generated the test, not which configuration resulted in that `t8n` being used.